hypervisor/lib/sprintf.c

```diff
@@ -515,19 +515,19 @@ int do_print(const char *fmt, struct print_param *param,
 						(param->vars.flags &
 						PRINT_FLAG_LONG_LONG) ?
 						__builtin_va_arg(args,
```

Comment:
For this one, it relies on int/long/long long... I am not sure if we should change.. Can you check with Jason? Same for others in __builtin_va_arg.